maven/templates/rules.bzl

```diff
@@ -171,7 +174,12 @@ def _deploy_maven_jar_impl(ctx):
     _generate_deployment_script(ctx, maven_coordinates)
 
     # there is also .source_jar which produces '.srcjar'
```

Comment:
what's the significance of this comment?

Reply:
@haikalpribadi this should give a hint why we use `.class_jar` in https://github.com/vmax/graknlabs-deployment/blob/master/maven/templates/rules.bzl#L174 as opposed to `.source_jar`
Any better way to phrase it?